src/Entity/Document.php

```diff
@@ -0,0 +1,170 @@
+ *     "title": "partial",
+ *     "visibility": "exact",
+ * })
+ *
+ * @ORM\Entity(repositoryClass="App\Repository\DocumentRepository");
+ * @ORM\Table(name="document")
+ *
+ * @UniqueEntity(fields={"title"}, message="document.title.unique_entity")
+ */
+class Document implements EntityAdministratorBlameableInterface
+{
+    use EntityIdentityTrait;
+    use EntityTimestampableTrait;
+    use EntityAdministratorBlameableTrait;
+
+    /**
+     * @ORM\Column
+     *
+     * @Assert\NotBlank(message="Veuillez renseigner un titre.")
+     * @Assert\Length(allowEmptyString=true, min=2, minMessage="Le titre doit faire au moins 2 caractères.")
+     *
+     * @Groups({"document_read"})
+     */
+    private ?string $title = null;
+
+    /**
+     * @ORM\Column(type="text", nullable=true)
+     *
+     * @Assert\Length(allowEmptyString=true, min=2, minMessage="La description doit faire au moins 2 caractères.")
+     *
+     * @Groups({"document_read"})
+     */
+    private ?string $comment = null;
+
+    /**
+     * @Assert\File(
+     *     maxSize="5M",
+     *     binaryFormat=false,
+     *     mimeTypes={
+     *         "image/*",
+     *         "video/mpeg",
+     *         "video/mp4",
+     *         "video/quicktime",
+     *         "video/webm",
+     *         "application/pdf",
+     *         "application/x-pdf",
+     *         "application/vnd.ms-powerpoint",
+     *         "application/vnd.openxmlformats-officedocument.presentationml.presentation",
+     *         "application/msword",
+     *         "application/vnd.ms-excel",
+     *         "application/vnd.openxmlformats-officedocument.wordprocessingml.document",
+     *         "application/vnd.openxmlformats-officedocument.spreadsheetml.sheet",
+     *         "application/rtf",
+     *         "text/plain",
+     *         "text/csv",
+     *         "text/html",
+     *         "text/calendar"
+     *     }
+     * )
+     */
+    private ?UploadedFile $file = null;
+
+    /**
+     * @ORM\Column(nullable=true)
+     */
+    private ?string $filePath = null;
+
+    public function __construct(UuidInterface $uuid = null)
+    {
+        $this->uuid = $uuid ?? Uuid::uuid4();
+    }
+
+    public function __toString()
+    {
+        return (string) $this->title;
+    }
+
+    public function getTitle(): ?string
+    {
+        return $this->title;
+    }
+
+    public function setTitle(string $title): void
+    {
+        $this->title = $title;
+    }
+
+    public function getComment(): ?string
+    {
+        return $this->comment;
+    }
+
+    public function setComment(?string $comment): void
+    {
+        $this->comment = $comment;
+    }
+
+    public function getFile(): ?UploadedFile
+    {
+        return $this->file;
+    }
+
+    public function setFile(?UploadedFile $file): void
+    {
+        $this->file = $file;
+    }
+
+    public function getFilePath(): ?string
+    {
+        return $this->filePath;
+    }
```

Comment:
what about removing all this getter/setter and make public properties with old good typehint ? I know it's crazy idea, but ...